packages/perseus-editor/src/widgets/label-image/select-image.jsx

```diff
@@ -3,15 +3,14 @@
  * Controlled component for selecting the question image, for answer labeling.
  */
 
+import {globalStyles} from "@khanacademy/perseus";
```

Comment:
Could you make a ticket to clean up (ie. delete) `globalStyles` and move to use WB stuff?  I'm not sure how big a task that is. I know in some cases a component uses a single color out of the styles, but some might be more complex.